src/App.tsx

```diff
@@ -1,14 +1,74 @@
+/* eslint-disable no-console */
 /* eslint-disable jsx-a11y/control-has-associated-label */
-import React from 'react';
+import React, { useEffect, useState } from 'react';
 import { UserWarning } from './UserWarning';
+import { getTodos } from './api/todos';
+import { Todo } from './types/Todo';
+import { TodoItem } from './components/Todo';
+import { Status } from './types/Status';
 
-const USER_ID = 0;
+const USER_ID = 12035;
 
 export const App: React.FC = () => {
+  const [todos, setTodos] = useState<Todo[]>([]);
+  const [filterByStatus, setFilterByStatus] = useState('all');
+
+  const [error, setError] = useState<null | string>(null);
+  const [showError, setShowError] = useState(false);
+
+  useEffect(() => {
+    const loadData = async () => {
+      setShowError(false);
+      setError(null);
+      getTodos(USER_ID)
+        .then((data) => {
+          setTodos(data);
```

Comment:
```suggestion
        .then(setTodos)
```

We can simplify it a bit